ansible/advanced.md

```diff
@@ -9,26 +9,62 @@ has_toc: false
 
 ## Advanced configuration
 
-### ii leader mode
-
-Ansible can be used as a leader device for the [ii](/docs/modular/ii)
-bus, controlling one or more follower modules using trigger and CV
-information sent from Ansible. Since Ansible is not wired to supply
-power to the ii bus, another leader module or a powered busboard is
-required to use Ansible as a leader. Having [Crow](/docs/crow) or
-[Teletype](../teletype/) connected to the same ii bus is sufficient.
-
-When Ansible is acting as a ii leader, it can no longer receive ii
-commands from other leaders like Teletype or Crow. Note also that
-sending messages from multiple leader devices simultaneously on the
-same ii bus does work in some tested scenarios, but may increase the
-risk of buggy behavior including modules locking up and needing a
-power cycle.
-
-Currently, the only interface for turning on leader mode uses a grid.
-In any grid application, go to the preset page by pressing the button
-next to the USB port. Follower device toggles are arranged vertically,
-just left of center. From top to bottom:
+  ###  what is i2c/ii?
```

Comment:
@mrmattallison , the indentation starting here breaks the markdown from being parsed correctly -- should resolve if you bring everything to the left :)

Reply:
Thanks @dndrks yip, resolved and working ace now.